WordPress/Classes/Extensions/WPStyleGuide+Search.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import WordPressShared
+
+extension WPStyleGuide {
+
+    public class func configureSearchBar(searchBar: UISearchBar) {
+        searchBar.backgroundColor = UIColor.blueColor()
+        searchBar.accessibilityIdentifier = NSLocalizedString("Search", comment: "")
+        searchBar.autocapitalizationType = .None
```

Comment:
I think you're right. I took this configuration code from the existing Themes search implementation, and haven't really re-considered the details. In fact, that background color is wrong too.

For the blog list and themes I think no autocorrection is the right way to go, as you're likely to be typing unusual names. For the post list, you're _probably_ going to be typing 'normal' words, so I might enable it there.